components/SearchSuggestions/SearchSuggestions.scss

```diff
@@ -68,6 +68,10 @@
 		padding: 0;
 		padding-bottom: 5px;
 
+		li.selected {
+			background-color: $gray-light;
```

Comment:
I imagine Vic will want the nav when it's completely finished to include the new UI Kit colors / fonts, etc. We can check with him on the details.